hbase-client/src/main/java/org/apache/hadoop/hbase/client/MetricsConnection.java

```diff
@@ -443,6 +447,15 @@ protected Ratio getRatio() {
     this.nsLookups = registry.counter(name(this.getClass(), NS_LOOKUPS, scope));
     this.nsLookupsFailed = registry.counter(name(this.getClass(), NS_LOOKUPS_FAILED, scope));
 
+    this.userRegionLockTimeoutCount =
+      registry.counter(name(this.getClass(), "userRegionLockTimeoutCount", scope));
+    this.userRegionLockWaitingTimer =
+      registry.timer(name(this.getClass(), "userRegionLockWaitingDurationMs", scope));
```

Comment:
While there is no universal practice or guideline here I don't think we should encode the time unit into the metric name. The unit for the metric can and should be documented. Once in place per operational compatibility guidelines it won't change. Making the unit in the name redundant.

Reply:
Sounds good to me. I have just updated by removing the `Ms` in the two new metrics I am adding in this PR. Thanks!